orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -331,3 +324,20 @@ def __setattr__(self, name: str, value) -> None:
         else:
             # Setting any new attributes should be done on ORTModule only when 'torch_module' is not defined
             self.__dict__[name] = value
+
+    def _switch_backend_to_pytorch(self, module):
```

Comment:
Instead of creating a method on `self`, shall we make this a `_util` method to any possibility of name conflicts with the users model.